### PR TITLE
#19: Move jbrowse config

### PR DESCRIPTION
**Pull request by samrichca**
- rename dms-jbrowse NPM library to dms-jbrowse-components
- add env variables for jbrowse human genome URLs
- move config related to loading files to DMS-UI

https://github.com/overture-stack/dms-jbrowse-components/pull/20

### CODE REVIEW

File: components/pages/explorer/Jbrowse/JbrowseWrapper.tsx [hunk 1]
```diff
@@ -201,12 +204,15 @@ const JbrowseWrapper = () => {
       ) : (
         <>
           <JbrowseLinear
+            assembly={jbrowseAssemblyObject}
+            assemblyName={jbrowseAssemblyName}
```
Comment (justincorrigible) [inline components/pages/explorer/Jbrowse/JbrowseWrapper.tsx:208]: my initial hunch is one is an Obj and one's a string... but what's the functional difference?

File: components/pages/explorer/Jbrowse/utils.tsx [hunk 2]
```diff
@@ -23,6 +23,9 @@ export const jbrowseAllowedFileTypes = ['BAM', 'VCF'];
 
 export const MAX_JBROWSE_FILES = 5;
 
+export const jbrowseAssemblyName = 'hg38';
+export const jbrowseAssemblyAlias = 'GRCh38';
+
```
Comment (justincorrigible) [inline components/pages/explorer/Jbrowse/utils.tsx:28]: looks to me like this file could be split into separate `helpers` and `contants`?

**Review by justincorrigible — APPROVED** [review]

File: components/pages/explorer/Jbrowse/JbrowseWrapper.tsx (hunk 1 above)
Comment (samrichca) [reply components/pages/explorer/Jbrowse/JbrowseWrapper.tsx:208]: one describes the entire assembly (i.e. which human genome version) and one is just the name in a string which is used in several places.

File: components/pages/explorer/Jbrowse/utils.tsx (hunk 2 above)
Comment (samrichca) [reply components/pages/explorer/Jbrowse/utils.tsx:28]: true, will do tomorrow

**Pull request merged**